guide.md

```diff
@@ -0,0 +1,136 @@
+# HIRO 7 JS Migration Guide (working draft)
+
+## Libraries
+
+### HIRO Graph Client
+
+`@hiro-graph/client@^7.3.9` (https://www.npmjs.com/package/@hiro-graph/client)
+
+```JS
+import Client from '@hiro-graph/client';
+
+const client = new Client({
+  endpoint: 'http://localhost:8888',
+  token: tokenInstance,
```

Comment:
This can be string or instance of Token class. Maybe add more info of it's api ?

Reply:
what would you put here?

Reply:
Ive added a bit about the token obect

Reply:
I just understood that it's possible to use the client not only on the client-side but also in node.js processes so getting token can vary depending on where it is used. So I think it may be enough with what you already added.